frontend/public/locales/en/translation.json

```
@@ -112,6 +112,10 @@
   },
   "tabsBox": {
     "errorOnLoading": "An error occurred while loading the results.",
+    "rateLater": "Your rate-later videos will appear here. You can add some to your list by clicking on the '+' sign on the video cards. You can also add them directly from the extension.",
+    "compared": "Compared videos will appear here.",
+    "recommendations": "Recommended videos will appear here.",
```

The "recommended videos" might be understood as the videos Tournesol recommend you to compare, instead of the videos the community is recommending.

As we may create a suggestions tab, which will contain videos Tournesol recommend you to compare, we can add few words to make the description more explicit.

```suggestion
    "recommendations": "Videos recommended by the community appear here.",
```

Moreover, I think we can use the indicative present. Instead of "will appear here", we can write simply "appear here".